src/client/utils/searchNarratives.ts

```diff
@@ -84,107 +113,110 @@ export default async function searchNarratives({
-      options.auth = false;
-      musts.push({
-        bool: {
-          should: [
-            { match: { narrative_title: 'tutorial' } },
-            { match: { narrative_title: 'narratorial' } },
-          ],
-        },
-      });
+      params.access = { only_public: true };
+      params.filters.fields.push({ field: 'is_narratorial', term: true });
       break;
     default:
       throw new Error('Unknown search category');
   }
 
   if (sort) {
     if (sort === 'Recently created') {
-      options.sort = [{ creation_date: { order: 'desc' } }, '_score'];
+      params.sorts = [
+        ['creation_date', SortDir.Desc],
+        ['_score', SortDir.Desc],
+      ];
     } else if (sort === 'Oldest') {
-      options.sort = [{ creation_date: { order: 'asc' } }, '_score'];
+      params.sorts = [
+        ['creation_date', SortDir.Asc],
+        ['_score', SortDir.Desc],
+      ];
     } else if (sort === 'Least recently updated') {
-      options.sort = [{ timestamp: { order: 'asc' } }, '_score'];
+      params.sorts = [
+        ['timestamp', SortDir.Asc],
+        ['_score', SortDir.Desc],
+      ];
     } else if (sort === 'Recently updated') {
-      options.sort = [{ timestamp: { order: 'desc' } }, '_score'];
+      params.sorts = [
+        ['timestamp', SortDir.Desc],
+        ['_score', SortDir.Desc],
+      ];
     } else {
       throw new Error('Unknown sorting method');
     }
   }
-  if (skip) {
-    options.skip = skip;
-  }
-  if (musts.length) {
-    options.query.bool.must = musts;
-  }
-  if (mustNots.length) {
-    options.query.bool.must_not = mustNots;
-  }
-  return await (await makeRequest(options)).result;
+  return await (await makeRequest(params)).result;
 }
 
 /**
  *
- * @param param0 SearchOptions - this takes a query, number of documents to skip,
+ * @param param0 SearchParams - this takes a query, number of documents to skip,
  *  sort parameter, auth (boolean, true if we're looking up personal data), and pageSize
  */
-async function makeRequest({
-  query,
-  skip = 0,
-  sort,
-  auth = true,
-  pageSize,
-}: SearchOptions): Promise<JSONRPCResponse> {
-  const headers = { 'Content-Type': 'application/json' };
-  if (auth) {
+async function makeRequest(params: SearchParams): Promise<JSONRPCResponse> {
+  const headers: { [key: string]: string } = {
+    'Content-Type': 'application/json',
+  };
+  if (!params.access || !params.access.only_public) {
+    // Requires an auth token
     const token = getToken();
     if (!token) {
       throw new Error(
         'Auth token not available for an authenticated search lookup!'
       );
     }
-    Object.assign(headers, { Authorization: token });
-  }
-  const params = {
-    indexes: [INDEX_NAME],
-    size: pageSize,
-    query,
-    from: skip,
-  };
-  if (sort) {
-    let sortObj = sort;
-    Object.assign(params, { sort: sortObj });
+    headers.Authorization = token;
   }
+  console.log('xyz headers', headers);
```

Comment:
Don't need console.log (I've been sloppy with it, too, admittedly...)

Reply:
will remove :p